### PR TITLE
Support for VersaDataset

### PR DESCRIPTION
**Pull request by mzuevx**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/__init__.py [hunk 1]
```diff
@@ -129,6 +129,7 @@
 from .gan_annotation_converter import GANAnnotationConverter
 from .kitti_converter import KITTIConverter
 from .smartlab_action_recognition import SmartLabActionRecognition
+from .versa import VersaDatasetConverter
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/__init__.py:132]: please rename file and class

**Comment by eaidova** [comment]
@mzuevx please resolve linting issues http://213.221.44.203/job/omz/job/precommit/job/accuracy_checker_ubuntu18/2924/artifact/pylint_out.log/*view*/

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        annotations = []
+        content_errors = []
+        meta = None
+
+        annotation = read_csv(self.annotation_file, is_dict=False)
+        data_dir = get_path(self.data_dir, is_directory=True)
+        known_dir = get_path(self.data_dir / self.known_folder, is_directory=True)
+        malicious_dir = get_path(self.data_dir / self.malicious_folder, is_directory=True)
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py:57]: could you please move paths configuration to configure method

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        for entry_id, (identifier, kind) in enumerate(annotation):
+            identifier = identifier.replace(':', '_')
+            kind = int(kind)
+            sample_path = malicious_dir / identifier if kind == 1 else known_dir / identifier
+            if check_content:
+                if not check_file_existence(sample_path):
+                    content_errors.append('{}: does not exist'.format(sample_path))
+            annotations.append(ClassificationAnnotation(str(Path(sample_path).relative_to(self.data_dir)), [kind,]))
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py:66]:
```suggestion
            annotations.append(ClassificationAnnotation(str(sample_path.relative_to(self.data_dir)), [kind,]))
```
above we have guarantees that sample_path is path object

Comment (eaidova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py:66]: also not sure that indention is correct, you add annotation only once, after cycle with samples processing

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/__init__.py (hunk 1 above)
Comment (mzuevx) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/__init__.py]: Fixed, with changing module name.

File: tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py (hunk 2 above)
Comment (mzuevx) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/versa.py]: fixed

**Pull request merged**